### ISSUE TITLE
CoffeeScript 1.11.0 is now available

### ISSUE THREAD

**Issue by lydell** [clutchski/coffeelint#580]
The test suite passes after upgrading (I just tried it out).

I didn’t make a pull request, because it don’t know if you’d like to take care of this first:

```
$ grep '1\.10\.0' -r src
src/rules/no_unnecessary_fat_arrows.coffee:                    # TODO: after <1.10.0 is not supported, remove child.isSuper
src/rules/duplicate_key.coffee:    # TODO: after <1.10.0 is not supported, remove 'IDENTIFIER' here
src/rules/duplicate_key.coffee:        # TODO: after <1.10.0 is not supported, remove 'IDENTIFIER' here
src/rules/no_debugger.coffee:    # TODO: after <1.10.0 is not supported, remove 'DEBUGGER' here
src/rules/no_stand_alone_at.coffee:        # TODO: after <1.10.0 is not supported, remove 'IDENTIFIER' here
src/rules/no_stand_alone_at.coffee:        # TODO: after v1.10.0 is not supported, remove 'IDENTIFIER' here
```

I also don’t know how the new `import`/`export` syntax will affect coffeelint.

**Comment by hildjj** [comment]
Take the opportunity to upgrade at least the glob version as well, if you decide to do a release. (I've seen #579)

**Comment by swang** [comment]
Published a v1.16.0
